benchmarks/perf-tool/okpt/test/steps/steps.py

```diff
@@ -690,6 +709,23 @@ def delete_model(endpoint, port, model_id):
     return response.json()
 
 
+def warmup_operation(endpoint, port, index):
+    """
+    Performs warmup operation on index to load native library files
+    of that index to reduce query latencies.
+    Args:
+        endpoint: Endpoint OpenSearch is running on
+        port: Port OpenSearch is running on
+        index: index name
+    Returns:
+        number of shards the plugin succeeded and failed to warm up.
+    """
+    response = requests.get('http://' + endpoint + ':' + str(port) +
```

Comment:
Seems we're using default timeout here, what is the chance that request will take longer than that time?

Reply:
I think our warmup API doesn't accept a timeout parameter. So, I think we can add a static sleep time of 5 min. WDYT ?

Reply:
that should work. We can do it as a parameter for this step, and use 5 mins as a default value

Reply:
As discussed offline, made the timeout parameter configurable to fix the issue